tests/unit/src/main/scala/scala/meta/internal/metals/WorkspaceSources.scala

```diff
@@ -1,73 +1,38 @@
 package scala.meta.internal.metals
 
-import java.nio.file.FileVisitResult
-import java.nio.file.Files
-import java.nio.file.Path
-import java.nio.file.SimpleFileVisitor
-import java.nio.file.attribute.BasicFileAttributes
-
 import scala.sys.process._
 import scala.util.control.NonFatal
 
 import scala.meta.internal.io.FileIO
 import scala.meta.internal.mtags.MtagsEnrichments._
 import scala.meta.io.AbsolutePath
 
+import geny.Generator
+
 /**
  * Finds sources in the workspace when there is no available build tool.
  */
 final class WorkspaceSources(workspace: AbsolutePath) {
-  def all: Traversable[AbsolutePath] = {
+  def all: Generator[AbsolutePath] = {
```

Comment:
I had some issues getting `walkFileTree` to work as an `Interable` in the same way that it is now. Then it sort of dawned on me that `all` here is only ever called in one place, inside of `indexWorkspace` which itself is only ever called in `WorkspaceFuzzBench.scala` and `BaseWorkspaceSymboleSuite.scala` to index a workspace with a zip it seems. I assumed we have the customer File Visitor to be able to skip dirs that start with `.` and not go down their subtrees, but seeing that this is only being used in tests it seemed _way way_ easier to instead just use the build in `listRecursive` on `AbsolutePath` and just deal with the Generator that is returned. Is this ok or am I missing something?